dask/dataframe/core.py

```diff
@@ -8013,9 +8014,11 @@ def maybe_shift_divisions(df, periods, freq):
 @wraps(pd.to_datetime)
 def to_datetime(arg, meta=None, **kwargs):
     tz_kwarg = {"tz": "utc"} if kwargs.get("utc") else {}
+
     if meta is None:
         if isinstance(arg, Index):
-            meta = pd.DatetimeIndex([], **tz_kwarg)
+            xd = get_meta_library(arg)
+            meta = xd.DatetimeIndex([], **tz_kwarg)
```

Comment:
Just a nit, but is there any good reason for creating the `xd` variable here (and below)?

Reply:
None that I can think of; will go with your below suggestion for both cases